doc/source/options.rst

```diff
@@ -31,7 +31,7 @@ Here is how you use ``html_theme_options`` to add breadcrumbs to the Ansys Sphin
 When you are on the module homepage, the breadcrumb displays the homepage
 title.  However, this title is not accessible to Sphinx from other
 documentation pages. Therefore, the ``html_short_title`` is used as the display
-text for the bread crumb on the documentation homepage. To ensure a consistent user
+text for the breadcrumb on the documentation homepage. To ensure a consistent user
```

Comment:
I think this word was splitted in an old PR, not really sure... @PipKat can provide more information about it.

Reply:
Yes, I wasn't sure because I saw both.

Reply:
@jorgepiloto @MaxJPRey 
It software, it is breadcrumb (1 word) in reference to navigation. Now, in food recipes, it would be two words!

Reply:
@PipKat Thanks for clarifying it.